### PR TITLE
Added osu!

### PR DESCRIPTION
**Pull request by Miko36110**
I have 0 coding experience so pls fix the errors, tried my best lol

# Proposed Changes

- added the rhythm game osu!

## Checklist

- [x] Tested locally
- [ ] Ran `yarn ci` to test my code `(don't know how to do that)`
- [x] Did not add any unnecessary changes
- [x] All my changes appear after other changes in each file
- [x] Included a screenshot (if adding a new button)
- [x] 🚀

alt text

### CODE REVIEW

File: src/components/Home/Home.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ import gitbucketLogo from '../../icons/gitbucket.svg';
 import shazamLogo from '../../icons/shazam.svg';
 import flickrLogo from '../../icons/flickr.svg';
 import tpdbLogo from '../../icons/tpdb.svg';
+import tpdbLogo from '../../icons/osu.svg';
```
Comment (Choubakawa) [inline src/components/Home/Home.js:73]: Duplicate of `tpdbLogo`, it needs to be named `osuLogo`.

File: src/components/Home/Home.js [hunk 2]
```diff
@@ -748,6 +749,15 @@ function Home(props) {
                   order={buttonOrder('TPDB')}
                 />
               )}
+              {runtimeConfig.OSU && (
+                <Button
+                  name="osu"
+                  href={runtimeConfig.OSU}
+                  displayName="osu!"
+                  logo={tpdbLogo}
```
Comment (Choubakawa) [inline src/components/Home/Home.js:757]: Have to change `tpdbLogo` to `osuLogo`

**Review by Choubakawa — CHANGES_REQUESTED** [review]
I find some lines that need to be changed but the rest looks good!

File: src/components/Home/Home.js (hunk 2 above)
Comment (Miko36110) [reply src/components/Home/Home.js:757]: haha thanks!

**Comment by timothystewart6** [comment]
Looks like there's some lint, otherwise it's good to go!

run `yarn lint --fix` locally and commit that and push it up!

**Comment by timothystewart6** [comment]
Thank you!

**Pull request merged**